cran-comments.md

```diff
@@ -0,0 +1 @@
+* With this submission, the maintainer has changed from Cole Arendt to Toph Allen.
```

Comment:
This is fine, but no one reads cran-comments.md. When you submit the next release, it will trigger an email to Cole to confirm the maintainer change, and that's it.

Reply:
Good to know!